lib/index.js

```diff
@@ -23,7 +25,15 @@ var Castle = module.exports = integration('Castle')
 Castle.prototype.initialize = function() {
   window._castle = window._castle || {};
   window._castle.q = window._castle.q || [];
-  window._castle.q.push(['setAppId', this.options.appId]);
+  window._castle.q.push(['setKey', this.options.publishableKey]);
+
+  if (this.options.cookieDomain) {
+    window._castle.q.push(['setCookieDomain', this.options.cookieDomain]);
+  }
+
+  if (this.options.autoPageview === false) {
+    window._castle.q.push(['autoTrack', this.options.autoPageview]);
```

Comment:
the segment snippet we tell people to use comes with a `.page()` call. So essentially each webpage load will result in a page call. So if you don't disable the autopage tracking by default, you will double count page views